### ISSUE TITLE
cgMLST.py database missing from install script

### ISSUE THREAD

**Issue by PyroEndy** [APHA-AMR-VIR/NewVMBasicProgInstall#3]
cgMLST.py requires database to be provided.

It's available here: https://bitbucket.org/genomicepidemiology/cgmlstfinder_db/src/master/

**Comment by jb2cool** [comment]
We will need to decide on an install location for these databases to live before be can include this maybe something like /opt/cgMLST/databases would be sensible

**Comment by PyroEndy** [comment]
Yes, the size of the database(s) are also an issue. For most people the ecoli one is probably only needed, however when I tried to run this on 2 isolates the program fell over. Could have been due to not enough RAM as I was running something else at the time.
